iothub/device/src/Transport/AmqpIot/AmqpIotSession.cs

```diff
@@ -221,7 +222,7 @@ internal bool IsClosing()
                 SndSettleMode = senderSettleMode,
                 RcvSettleMode = receiverSettleMode,
             };
-            amqpLinkSettings.AddProperty(AmqpIotErrorAdapter.TimeoutName, timeout.TotalMilliseconds);
+            //amqpLinkSettings.AddProperty(AmqpIotErrorAdapter.TimeoutName, timeout.TotalMilliseconds);
```

Comment:
intention?

Reply:
ah ! thanks for noticing, I need to confirm whether or not we can get rid of this property or not. I will add a todo

Reply:
```suggestion
            //TODO: azabbasi what is this property, do we need it?
            //amqpLinkSettings.AddProperty(AmqpIotErrorAdapter.TimeoutName, timeout.TotalMilliseconds);
```